app/processing/delta/split-to-ledger.js

```diff
@@ -3,16 +3,16 @@ const { AP } = require('../../ledgers')
 const ensureValueConsistency = require('./ensure-value-consistency')
 const { v4: uuidv4 } = require('uuid')
 
-const splitToLedger = (paymentRequest, unsettledValue, unsettledLedger) => {
+const splitToLedger = (paymentRequest, targetValue, ledger) => {
```

Comment:
renaming variables from `unsettled` as that is misleading when it comes to AR ledger